bodhi/tests/client/test___init__.py

```diff
@@ -1659,6 +1694,26 @@ def test_info_with_errors(self, send_request):
         self.assertEqual(result.exit_code, 1)
         self.assertEqual(result.output, ("ERROR: an error was encountered... :(\n"))
 
+    @mock.patch('bodhi.client.bindings.BodhiClient.csrf',
+                mock.MagicMock(return_value='a_csrf_token'))
+    @mock.patch('bodhi.client.bindings.BodhiClient.send_request',
+                return_value=client_test_data.EXAMPLE_INFO_RELEASE_MUNCH, autospec=True)
+    def test_verbose_flag(self, send_request):
+        """
+        Assert correct behavior with the --verbose flag.
+        """
+        runner = testing.CliRunner()
+
+        result = runner.invoke(client.info_release, ['--url', 'http://localhost:6543', 'F27',
+                                                     '--verbose'])
+
+        self.assertEqual(result.exit_code, 0)
+        self.assertEqual(result.output,
+                         client_test_data.EXPECTED_INFO_RELEASE_OUTPUT)
```

Comment:
Since ```EXAMPLE_INFO_RELEASE_MUNCH``` does not have critpath packages in it, this test does not assert that they are rendered in the output.

Reply:
Ah yes, I had missed that. Nevermind then.